server/data/data-cassandra/src/main/java/org/apache/james/user/cassandra/CassandraUsersRepositoryModule.java

```diff
@@ -44,7 +44,9 @@ public CassandraUsersRepositoryModule() {
                         .addPartitionKey(CassandraUserTable.NAME, text())
                         .addColumn(CassandraUserTable.REALNAME, text())
                         .addColumn(CassandraUserTable.PASSWORD, text())
-                        .addColumn(CassandraUserTable.ALGORITHM, text())));
+                        .addColumn(CassandraUserTable.ALGORITHM, text())
+                        .withOptions()
+                        .comment("Stores user of this James server (if applicable)")));
```

Comment:
I meant "if no LDAP", but this table will not be present due to our nice orthogonal architecture on cassandra-guice-ldap.

Hence I removed it.